roles/slurm-management/tasks/main.yml

```diff
@@ -130,10 +130,10 @@
     login_password: "{{ MYSQL_ROOT_PASSWORD }}"
     name: "{{ slurm_storage_user }}"
     password: "{{ slurm_storage_pass }}"
-    host: '%'
+    host: 'localhost'
     priv: "{{ slurm_database_name }}.*:ALL"
     connect_timeout: 120
-  no_log: True
+#  no_log: True
```

Comment:
Fine for debugging, but better to re-enable the the ```no_log: True``` before merging and applying this to production ;)